### PR TITLE
Update tests for command consolidation

### PR DESCRIPTION
**Pull request by openbots-su**

### CODE REVIEW

**Review by openbots-ff — CHANGES_REQUESTED** [review]
In addition to fixing the test references, you also need to consolidate them as well.

File: OpenBots.Commands.Test/OpenBots.Commands.DataManipulation.Test/OpenBots.Commands.DataManipulation.Test.csproj [hunk 1]
```diff
@@ -35,18 +35,12 @@
...
     <Reference Include="BouncyCastle.Crypto, Version=1.8.9.0, Culture=neutral, PublicKeyToken=0e99375e54769942, processorArchitecture=MSIL">
       <HintPath>..\..\packages\Portable.BouncyCastle.1.8.9\lib\net40\BouncyCastle.Crypto.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Bcl.AsyncInterfaces, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Bcl.AsyncInterfaces.5.0.0\lib\net461\Microsoft.Bcl.AsyncInterfaces.dll</HintPath>
-    </Reference>
     <Reference Include="Microsoft.Office.Interop.Outlook, Version=15.0.0.0, Culture=neutral, PublicKeyToken=71e9bce111e9429c, processorArchitecture=MSIL">
       <HintPath>..\..\packages\Microsoft.Office.Interop.Outlook.15.0.4797.1003\lib\net20\Microsoft.Office.Interop.Outlook.dll</HintPath>
-      <EmbedInteropTypes>False</EmbedInteropTypes>
+      <EmbedInteropTypes>True</EmbedInteropTypes>
```
Comment (openbots-ff) [inline OpenBots.Commands.Test/OpenBots.Commands.DataManipulation.Test/OpenBots.Commands.DataManipulation.Test.csproj:43]: you should go through all the outlook references in each of the the projects where it exists and set the "Embed Interop Types" to false

**Review by openbots-ff — CHANGES_REQUESTED** [review]
image
OpenBots.Command.Core.Test is missing an 's' in Commands and DataBase should be Database

**Comment by openbots-ff** [comment]
I also noticed that the Database test project doesn't have any actual commands in it

**Review by openbots-ff — CHANGES_REQUESTED** [review]
It's not enough to just rename the project. You also have to rename the file directory, the AssemblyInfo.cs metadata, and the RootNamespace/AssemblyName in the .csproj.

File: OpenBots.Commands.Test/OpenBots.Commands.DataManipulation.Test/OpenBots.Commands.DataManipulation.Test.csproj (hunk 1 above)
Comment (openbots-su) [reply OpenBots.Commands.Test/OpenBots.Commands.DataManipulation.Test/OpenBots.Commands.DataManipulation.Test.csproj]: Addressed in b3ebf74

**Review by openbots-ff — APPROVED** [review]

**Review by openbots-ff — DISMISSED** [review]

**Review by openbots-ch — APPROVED** [review]

**Review by openbots-ch — APPROVED** [review]

**Review by openbots-ff — APPROVED** [review]

**Pull request merged**